.github/workflows/ci.yml

```diff
@@ -204,6 +204,20 @@ jobs:
       - name: 'Check working directory status'
         run: './scripts/check-git-status.sh'
 
+  cargo_test:
+    name: Run cargo test (excluding relay tests)
```

Comment:
How's the name? Happy to tweak it!

We could also run it as part of the `rustfmt` just to consolidate the tooltchain download stuff. Would shave some seconds off the run time.

Reply:
Name SGTM, also keeping it a separate step sounds good to me. I think it's fine, because these steps are run in parallel and aren't likely to slow down the `all-checks-passed` step, so this is great.